requirements.txt

```diff
@@ -0,0 +1,6 @@
+netCDF4
+numpy
+pandas
```

Comment:
pysat imposes minimum versions on both of these.  Not sure if we need to do that here as well.

Reply:
I believe the pysat install will take care of that.